definition/data/sheets/CaseEvent.json

```diff
@@ -14,7 +14,7 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "ClaimantAccepts", "Name": "Claimant accepted", "Description": "Claimant Accepts", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ClaimantRejects", "Name": "Claimant rejected", "Description": "Claimant Rejects", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "AgreementSignedByClaimant", "Name": "Claimant signed SA", "Description": "Claimant Signs Settlement", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "RejectOrganisationPaymentPlan", "Name": "Plan proposed for Org", "Description": "New plan proposed for organisation", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "RejectOrganisationPaymentPlan", "Name": "Plan proposed for Org", "Description": "New plan proposed for organisation", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;judgmentDecideAmount", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": "5,10,15"},
```

Comment:
I think it is preferable to disable retries for now at least until we identify a clear need/benefit. Just as always concerned about side effects.

Reply:
Done